backend/api/projects.py

```diff
@@ -108,6 +108,30 @@ def get_projects():
     return res
 
 
+@projects.route('/api/projects/authors', methods=['GET'])
+@login_required
+def get_all_authors():
+    try:
+        authors = g.projects.distinct('authors')
+        if not authors:
+            return jsonify([])
+        result = sorted([author['name'] for author in authors] +
+                        [author['email'] for author in authors], key=str.lower)
```

Comment:
shouldn't we return a list of author dicts instead? I mean we can sort them this way, but returning names first and emails after with no real distinction doesn't seem a good idea